### PR TITLE
Add capnp.json manifest to Main bucket

### PR DESCRIPTION
**Pull request by GioPat**
Adds Cap'n protocol to main bucket

Create manifest, with the autoupdate, of the [Cap'n Proto](https://capnproto.org/).

- [x] I have read the [Contributing Guide](https://github.com/ScoopInstaller/.github/blob/main/.github/CONTRIBUTING.md).

### CODE REVIEW

**Comment by GioPat** [comment]
Can someone review this PR?

File: bucket/capnp.json [hunk 1]
```diff
@@ -0,0 +1,25 @@
+{
+    "version": "1.0.2",
+    "description": "Capnp Proto is an insanely fast data interchange format and capability-based RPC system",
```
Comment (tech189) [inline bucket/capnp.json:3]:
```suggestion
    "description": "Cap'n Proto is an insanely fast data interchange format and capability-based RPC system",
```
Suggestion to fix the name in the description

File: bucket/capnp.json [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+    "url": "https://capnproto.org/capnproto-c++-win32-1.0.2.zip",
```
Comment (tech189) [inline bucket/capnp.json:4]: Could you please move the `url` and `hash` to the correct place as per the order here https://github.com/ScoopInstaller/.github/blob/main/.github/CONTRIBUTING.md#for-scoop-buckets ?

**Review by tech189 — CHANGES_REQUESTED** [review]

File: bucket/capnp.json (hunk 2 above)
Comment (GioPat) [reply bucket/capnp.json]: Done.

**Comment by GioPat** [comment]
Can we now close this? @tech189

File: bucket/capnp.json (hunk 2 above)
Comment (tech189) [reply bucket/capnp.json]: Sorry I just meant putting `url` and `hash` after `license`, you don't need `architecture` if the app is 32bit only.

**Review by tech189 — APPROVED** [review]

**Comment by tech189** [comment]
All done now, thank you for your contribution @GioPat!

**Pull request merged**